R/calc_weights_and_outputs.R

```diff
@@ -207,6 +212,7 @@ calc_weights_and_outputs <- function(
     )
   }
 
+  return(NULL)
```

Comment:
While I prefer explicitly returning whatever like this, linters (following tidyverse style) usually tell you to just pass the object

Reply:
Yeah, checking for explicit return is one of the changes in the `.lintr` file I made for this repo.